test/component/applicant/testAliasReason.js

```diff
@@ -26,10 +26,12 @@ describe('applicant-alias-reason', () => {
                 }
             };
 
+            const contentToExclude = ['reasonForNameChangeQuestionSummary', 'optionDifferentSpelling'];
```

Comment:
Added 'optionDifferentSpelling' to content to exclude. Happy to look into if incorrect.